File: src/main/kotlin/com/demo/models/Address.java
package com.demo.models;

import javax.persistence.*;

@Entity
public class Address {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false, updatable = false)
    public Long id;

    @Column(nullable = false)
    public String address;

}

File: src/main/kotlin/com/demo/models/Address.kt
package com.demo.models

import javax.persistence.*

@Entity
class Address() {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "id", nullable = false, updatable = false)
    var id: Long = 1

    @Column(nullable = false)
    var address: String = ""

    constructor(address: String) : this() {
        this.address = address
    }

}